extensions/wikia/AutomaticWikiAdoption/AutomaticWikiAdoptionHelper.class.php

```diff
@@ -190,7 +190,7 @@ static function adoptWiki($wikiId, $user) {
 					//log
 					self::addLogEntry($admin, $oldGroups, $newGroups);
 					//Unset preference for receiving future adoption emails
-					$admin->setLocalPreference( "adoptionmails", $wikiId, 0 );
+					$admin->setLocalPreference( "adoptionmails", 0, $wikiId );
```

Comment:
yeah, i found it when i was looking for usages to see if any other place was omitting the wikiId